reconcile/saas_file_owners.py

```diff
@@ -377,10 +378,16 @@ def run(
             gl.remove_label_from_merge_request(gitlab_merge_request_id, HOLD)
         if not valid_lgtm:
             gl.remove_label_from_merge_request(gitlab_merge_request_id, APPROVED)
-            comment_line_body = (
-                f"- changes to saas file '{saas_file_name}' "
-                + f"require approval (`/lgtm`) from one of: {saas_file_owners}."
-            )
+            if diff["deprecated"]:
+                comment_line_body = (
+                    f"- saas file '{saas_file_name}' is deprecated. "
+                    + "please migrate resources to a different saas file."
+                )
```

Comment:
I would like to see the '/lgtm' message as well, maybe something like that:

```
- changes to saas file '{saas_file_name}' ...
- [bold red]saas file 'XXX' is deprecated. Please migrate resources to a different saas file.[/]
```

Reply:
otherwise is hard to see, that we can still `/lgtm` that

Reply:
if the owners of the saas file decided it's deprecated, why should their names be shown? :sweat_smile:

Reply:
removing this part from this PR